gatsby/src/docs/guides/alert-notifications/creating-custom-rules.mdx

```diff
@@ -0,0 +1,118 @@
+
+Alert rules consist of:
+
+- A set of self-explanatory `Conditions` connected in an _ALL_, _ANY_, or _NONE_ relation.
+- The `Environment` where your code is deployed and running in which you'd like the alert rule applied.
+- The `Actions` that should be taken when the associated conditions are met. Mainly -- where you'd like to channel this alert.
+
+#### **Creating a good set of alert rules means:**
+
+1. Identifying the critical show-stopping errors and routing them appropriately (to PagerDuty for instance) so they can be resolved by the right people as soon as possible.
+
+2. Maintaining awareness and visibility to all other errors, in terms of where the alerts are routed to (mail, specific Slack channels) and at what frequency.
+
+Generally, we recommend you configure these rules and fine-tune them as you go, adapting to your teams workflows and preferences. With that said, there are some common best practices that you should consider.
+
+## Selecting Conditions
+
+The `New Alert Rule` wizard allows you to compose alert rules combining multiple conditions. These conditions are pretty self explanatory and rely on event `Tags` and `Attributes` values, event `Frequency`, and `Issue State` changes. Essentially, allowing you to combine the conditions bellow to fit your specific use-case.
+
+![Alert Rule Conditions](alerts_notifications_09.png)
+
+### **Event Tags**
+
+Tags are key-value pairs that Sentry assigns to each event. Some of the tags are added by default depending on the platform and type of SDK. Developers can also add **Custom Tags** through the SDK. You can find the list of tags available in your project under `[Project Settings] > Tags`. The list is an aggregation of all of the tag keys (default and custom) that have been encountered in events in this project. For more information take a look at [Tagging Events](https://docs.sentry.io/enriching-error-data/additional-data/?platform=browser#tagging-events)
+
+Event tags are useful for various reasons. First, tags are indexed and this allows you to query your errors in the Sentry `Event Stream` and `Issue Stream` based on specific tag values. In addition, adding the right tags in your code will allow you tell Sentry to notify you only when events with specific details occur.
+
+To create a rule based on a tag's value, select the condition
+
+```
+An event's tags match {key} {match} {value}
+ ```
+
+ A tag-based condition can be used as the sole condition for an alert rule or in conjunction with additional conditions to fine tune the rule:
+
+![Alert Rule Event Tags](alerts_notifications_11.png)
+> In this rule we're leveraging custom tags that our developers added through the Sentry SDK, telling Sentry to notify us via our _#sentry-urgent_ **Slack** channel when _Enterprise_ customers are experiencing errors in critical parts of our application.
```

Comment:
out of habit im almost always put newlines between blocks just to ensure it doesnt do something weird. i haven't used this syntax before though so as long as you verified its rendering it should be fine.